src/EventStore.Projections.Core.Javascript.Tests/EventStore.Projections.Core.Javascript.Tests.csproj

```diff
@@ -30,13 +30,13 @@
 
   <ItemGroup>
 		<PackageReference Include="GitHubActionsTestLogger" Version="2.0.1" />
-		<PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.9.1" />
-    <PackageReference Include="xunit" Version="2.4.1" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3">
+		<PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.2" />
```

Comment:
fix whitespace (for previous line as well)

Reply:
since the whitespace was pre-existing im inclined to leave it for now and swing back round on csproj formatting in another pr, there are other places where they're a bit janky too